.gitignore

```diff
@@ -14,3 +14,4 @@ node_modules
 lib
 test-results
 *.generated.js
+test.html
```

Comment:
I think adding `test.html` to `.gitignore` is redundant?

Reply:
I agree. It wasn’t my intention to upload that. I made a local file to help me in the process. I will fix it.